waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -190,15 +191,15 @@ method register*(
   g.registrationTxHash = some(txHash)
   # the receipt topic holds the hash of signature of the raised events
   # TODO: make this robust. search within the event list for the event
-  debug "ts receipt", tsReceipt
+  # debug "ts receipt", tsReceipt
```

Comment:
Why?

Reply:
Sure, fixed. ReceipObject changed to ref object...